model-optimizer/mo/utils/custom_replacement_config.py

```diff
@@ -349,6 +352,17 @@ def parse_custom_replacement_config_file(file_name: str):
         raise Error("Failed to parse custom replacements configuration file '{}': {}. ".format(file_name, exc) +
                     refer_to_faq_msg(70)) from exc
 
+    try:
+        base_dir = get_mo_root_dir()
+        schema_file = os.path.join(base_dir, 'mo', 'utils', 'schema.json')
+        with open(schema_file, 'r') as f:
+            schema = json.load(f)
+            validator = json_validate.compile(schema)
+            validator(data)
+    except Exception as exc:
+        raise Error("Failed to validate custom replacements configuration file '{}': {}. ".format(file_name, exc) +
+                    refer_to_faq_msg(70)) from exc
```

Comment:
I propose to move this chunk into a separate function sort of `validate_json_config`. And you will test this function in unit tests instead of copy-paste  in each test.